Add --ensure_metadata --urls to planemo shed_lint. Fix many .shed.yml files

nsoranzo opened the pull request.

nsoranzo commented:
I may need to break up this PR because it produces too long log files and Travis kills the builds with:
```
The log length has exceeded the limit of 4 Megabytes (this usually means that test suite is raising the same exception over and over).

The build has been terminated.
```

yhoogstrate commented:
Great work @nsoranzo, could you also fix this one: https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_picard_2_0_1/.shed.yml

bgruening commented:
@nsoranzo sorry the travis config has changed. Can you please rebase it against master?

nsoranzo commented:
@yhoogstrate I've fixed `packages/package_picard_2_0_1/.shed.yml` as you asked.
@bgruening I've rebased over latest master.

Because of the log length limit, I've reduced this PR to fix only data managers, a few packages and a couple of tools, will open more PRs after this is merged.

bgruening commented:
Thanks @nsoranzo!
